.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "semi": true,
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "printWidth": 120,
+  "useTabs": false,
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "arrowParens": "avoid",
+  "endOfLine": "lf"
+}
```

Comment:
Is there a prettier rule on import order?

The style I took from Python developers is:

```
Built-in module imports

Third-party module imports

Project relative path imports
```

with each block sorted alphabetically.

Reply:
Oh this would be nice, I did not see it in the docs, but maybe there's a third-party plugin for it